docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -76,9 +76,9 @@ The Telemetry Operator watches all LogPipeline resources and related Secrets. Wh
     ```
     An output is a data destination configured by a [Fluent Bit output](https://docs.fluentbit.io/manual/pipeline/outputs) of the relevant type. The LogPipeline supports the following output types:
 
-    - **http**, which pushes the data to the specified http destination.
-    - **grafana-loki**, which pushes the data to the Kyma-internal Loki instance. Note: This output might not be compatible with latest Loki versions. For that, use the `custom` output with name `loki` instead.
-    - **custom**, which supports the configuration of any destination in the Fluent Bit configuration syntax.
+    - **http**, which sends the data to the specified http destination. The output is designed to integrate with an [FluentD HTTP Input](https://docs.fluentd.org/input/http) which opens up a huge ecosystem of integration possibilities.
+    - **grafana-loki**, which sends the data to the Kyma-internal Loki instance. Note: this output is considered legacy and is only available for backwards compatibility with the in-cluster Loki instance. It might not be compatible with latest Loki versions. For integration with Loki, use the `custom` output with name `loki` instead.
+    - **custom**, which supports the configuration of any destination in the Fluent Bit configuration syntax. Note: Usage of a `custom` output will put the LogPipeline in [unsupported mode](#unsupported-mode).
```

Comment:
I suggest to put this as the 2nd bullet and `grafana-loki` as 3rd, considering a) `grafana-loki` mentions `custom`, so it `custom` should be introduced earlier, and b) `grafana-loki` is legacy and thus should come last.

```suggestion
    - **custom**, which supports the configuration of any destination in the Fluent Bit configuration syntax. Note: If you use a `custom` output, you put the LogPipeline in [unsupported mode](#unsupported-mode).
```